apps/browser/src/autofill/services/autofill.service.ts

```diff
@@ -234,7 +236,11 @@ export default class AutofillService implements AutofillServiceInterface {
       }
     }
 
-    if (cipher == null || cipher.reprompt !== CipherRepromptType.None) {
+    if (
+      cipher == null ||
+      (cipher.reprompt !== CipherRepromptType.None &&
+        (await this.userVerificationService.hasMasterPasswordAndMasterKeyHash()))
+    ) {
```

Comment:
conflict resolution - we removed the null check in https://github.com/bitwarden/clients/pull/5621/files#diff-ff919521412a590452c7e201f94d1c80997766b29cf4870faa84275768edc6a8L237

```suggestion
    if (
      cipher.reprompt !== CipherRepromptType.None &&
      await this.userVerificationService.hasMasterPasswordAndMasterKeyHash()
    ) {
```

Reply:
Ty for catching these! Resolved w/ #6042!